src/app/services/fiddle.service.spec.ts

```diff
@@ -18,9 +18,10 @@ describe('fiddle.service', () => {
     FiddleService
   ]);
   
-  beforeEach(inject([FiddleService], s => {
-    fiddle = s;
-  }));
+  // would like to get rid of redundant inject, but this is giving a ts error.
```

Comment:
This became more important than just redundancy because I cannot seem to find a pattern for use with testing async calls.